test/e2e_federation/util.go

```diff
@@ -141,21 +141,71 @@ func createService(clientset *fedclientset.Clientset, namespace, name string) (*
 	}
 
 	By(fmt.Sprintf("Trying to create service %q in namespace %q", service.Name, namespace))
-	return clientset.CoreV1().Services(namespace).Create(service)
+	var err error
```

Comment:
This block may not be required and the original one should suffice. Can you please revert this one and also preserve the signature of createService function